src/yetibot/core/commands/collections.clj

```diff
@@ -124,7 +129,7 @@
                      [args (merge cmd-params {:raw item :opts item})]
                      [(psuedo-format args item) (merge cmd-params {:raw item :opts nil})]))
             (catch Exception ex
-              (error "Exception in xargs pmap"
+              (error "Exception in xargs cmd-runner"
```

Comment:
We could tell the user which function was used as the `cmd-runner`.

You can get the name if you refer to the vars above in line 119 instead of passing the function itself, e.g.:

```clojure
(if (contains? cat :async) #'map #'pmap)
```

This won't affect the execution, but it allows us to get at the meta:

```clojure
(error "Exception in xargs cmd-runner" (-> cmd-runner meta :name) 
  ...)
```

I tested this in the REPL with a standalone example:

```clojure
(let [f #'map]
  [(:name (meta f))
   (f inc (range 3))])
```

Reply:
Thanks for the suggestion here!  I had wanted to improve this but was hitting a wall.  Your suggestion got me pointed in the right direction.  I took a slightly different approach... let me know what you think.

Here are two log scraps (I was logging at `INFO` during dev, to keep my console noise down, but it's back to `DEBUG` now):

```
18-10-26 11:22:48 elder-whale INFO [yetibot.core.commands.collections:11] - xargs using cmd-runner: pmap for command "echo hi:"                                             

18-10-26 11:24:41 elder-whale INFO [yetibot.core.commands.collections:11] - xargs using cmd-runner: map for command "replyto @U3CTS2KG8 "                                   
```

Reply:
Looks good!